products.json

```diff
@@ -1699,9 +1699,643 @@
             1500,
             9000
           ],
-          "extended_multizone": true,
-          "upgrades": []
-        }
+          "extended_multizone": true
+        },
+        "upgrades": []
+      },
+      {
+        "pid": 145,
+        "name": "LIFX Ceiling US",
+        "features": {
+          "color": true,
+          "chain": false,
+          "matrix": true,
+          "infrared": false,
+          "multizone": false,
+          "temperature_range": [
+            1500,
+            9000
+          ]
+        },
+        "upgrades": []
+      },
+      {
+        "pid": 146,
+        "name": "LIFX Ceiling Intl",
```

Comment:
This is capitalised later, probably want to stick to `Intl` to be consistent as it's not an acronym

Reply:
Good spotting, will fix